src/targets/targets.service.ts

```diff
@@ -38,4 +38,9 @@ export class TargetsService {
     const target = new Target(title, radius, latitude, longitude, user, topic)
     return new TargetDto(await this.targetsRepository.save(target))
   }
+
+  async findByUser(userInfo) {
```

Comment:
Can we use a type here to see what it would be? `userInfo` seems a bit vague. Maybe an interface would help

Reply:
Plus you are allowing the user to access your database directly by not whitelisting

Reply:
User can be an object or an integer, to find by id.
The user does not have access to this method. Right is being called with the user form the request. If you mean by messing with the JWT noone should be able to generate one without the secret.